modules/apps/portal-url-builder/portal-url-builder-api/src/main/java/com/liferay/portal/url/builder/APIRequestAbsolutePortalURLBuilder.java

```diff
@@ -14,12 +14,18 @@
 
 package com.liferay.portal.url.builder;
 
+import com.liferay.portal.url.builder.facet.BuildableAbsolutePortalURLBuilder;
+
 /**
- * Builds a module resource URL. Module resources live in {@code
- * com.liferay.portal.kernel.util.Portal#getPathModule()}.
+ * Builds a URL that connects to a dynamic servlet.
+ *
+ * <p>
+ * API requests are, by definition, not cacheable and should not be stored in
+ * CDNs.
+ * </p>
  *
  * @author Iván Zaera Avellón
  */
-public interface ModuleAbsolutePortalURLBuilder
+public interface APIRequestAbsolutePortalURLBuilder
```

Comment:
Not sure about the scope of this class name. It may be misleading if one thinks that there exists headless API urls.

Reply:
I also though about that, but couldn't find a better name... Maybe something like `ServletRequest`, `EndpointRequest`, `ServletServiceRequest`, .... :thinking: 

Ideas?

Reply:
Let's call it `ServletAbsolutePortalURLBuilder`...